mogc_partnerships/pipeline.py

```diff
@@ -18,7 +18,7 @@ def run_filter(self, user, course_key, mode):
             cohort_ids = CohortOffering.objects.filter(offering=offering).values_list(
                 "cohort_id", flat=True
             )
-            if user.memberships.filter(cohort__in=cohort_ids).exists():
+            if user.memberships.filter(cohort__in=cohort_ids).exists() or user.is_staff:
```

Comment:
Instead of `user.is_staff`, which describes a platform-wide role, we'll need to determine if the user has a `PartnerManagementMembership` for the partnership associated with the course.